src/web/mjs/connectors/CopyToon.mjs

```diff
@@ -0,0 +1,43 @@
+import GnuBoard5BootstrapBasic2 from './templates/GnuBoard5BootstrapBasic2.mjs';
+
+export default class CopyToon extends GnuBoard5BootstrapBasic2 {
+
+    constructor() {
+        super();
+        super.id = 'copytoon';
+        super.label = 'CopyToon';
+        this.tags = [ 'webtoon', 'korean' ];
+        this.url = 'https://copytoon88.com';
+
+        this.path = [ '/웹툰', '/포토툰' ];
+        this.scriptPages = `
+            new Promise(resolve => {
+                resolve([...document.querySelectorAll('div#bo_v_con img')].map(img => img.src));
+            });
+        `;
+
+        this._initializeConnector();
+    }
+
+    async _initializeConnector() {
+        let uri = new URL(this.url);
+        uri.searchParams.set('ts', Date.now());
+        uri.searchParams.set('rd', Math.random());
+        let request = new Request(uri.href, this.requestOptions);
+        await Engine.Request.fetchUI(request, '');
```

Comment:
`fetchUI` seems unused

Reply:
It will bypass CloudFlare challenge (if needed) and set all the corresponding cookies for all upcoming requests to this website. Most websites may not need this, but for consistency it is done on every website (usually in the base class, but the method is overwritten for the upcoming domain detection from twitter)